src/Aspire.Hosting.Dapr/CommandLineBuilder.cs

```diff
@@ -130,6 +168,16 @@ private static CommandLineArgBuilder NamedStringArg(string name, string value, b
         };
     }
 
+    private static CommandLineArgBuilder ModelNamedStringArg(string name, string value, bool assignValue)
+    {
+        return () =>
+        {
+            return assignValue
+                ? new[] { $"{name}={value}" }
```

Comment:
Are reserved characters not an issue in this case?  I'm not sure I follow the differences here.

Reply:
Ok, how about whitespace?

Reply:
Same thing. It is an API call; the values on both sides of the wire are/should be exactly the same.